lib/pxutil.js

```diff
@@ -205,6 +206,24 @@ function parseCookieHeader(headerValue) {
     return res;
 }
 
+function accessNestedObjectValueByStringPath(path, obj, separator='.') {
+    var properties = Array.isArray(path) ? path : path.split(separator)
```

Comment:
why do you call this properties? isn't it just _pathAsArray_?

Reply:
They're the nested properties of the object